.ruby-version

```diff
@@ -1 +1 @@
-2.6.6
```

Comment:
So from what I understand this version is used for local development/ci/doesn't affect downstream users which may have different versions of rails/ruby running